.github/labeler.yml

```diff
@@ -72,6 +72,10 @@
 - 'samples/cpp/visual_language_chat/**/*'
 - 'samples/python/visual_language_chat/**/*'
 
+'category: RAG samples':
+- 'samples/cpp/rag/**/*'
+- 'samples/python/rag/**/*'
```

Comment:
looks like we need to add test files (like tests/python_tests/samples/test_text_embedding_pipeline.py) to samples category as well to minimize having `no-match-files`

Reply:
`tests/python_tests/samples/test_text_embedding_pipeline.py` added to category. I'll add the rest of samples tests to the appropriate categories in the next PR

Reply:
Fixed here for other categories https://github.com/openvinotoolkit/openvino.genai/pull/2168